include/dxc/dxcapi.internal.h

```diff
@@ -99,7 +99,8 @@ enum LEGAL_INTRINSIC_COMPTYPES {
   LICOMPTYPE_UINT8_4PACKED = 41,
   LICOMPTYPE_ANY_INT16_OR_32 = 42,
   LICOMPTYPE_SINT16_OR_32_ONLY = 43,
-  LICOMPTYPE_COUNT = 44
+  LICOMPTYPE_ANY_SAMPLER = 44,
+  LICOMPTYPE_COUNT = 45
```

Comment:
Is there a specific reason why LICOMPTYPE_COUNT is now assigned to 45 instead of leaving it at 44 and letting LICOMPTYPE_ANY_SAMPLER taking the 45 spot?